### ISSUE TITLE
How to get SiteID and SiteURL from the sp.site.createCommunicationSite ?

### ISSUE THREAD

**Issue by jefflundberg** [pnp/pnpjs#827]
### Category
- [ ] Enhancement
- [ ] Bug
- [x] Question
- [ ] Documentation gap/issue

### Version

Please specify what version of the library you are using: [1.3.5]

Please specify what version(s) of SharePoint you are targeting: [SharePoint Online]

### Question
How does one get the SiteID and URL from the ```sp.site.createCommunicationSite``` command?

From the example in the documentation:

```typescript
import { sp } from "@pnp/sp";

const s = await sp.site.createCommunicationSite(
            "Title",
            1033,
            true,
            "https://tenant.sharepoint.com/sites/commSite",
            "Description",
            "HBI",
            "f6cc5403-0d63-442e-96c0-285923709ffc",
            "a00ec589-ea9f-4dba-a34e-67e78d41e509",
            "user@TENANT.onmicrosoft.com");
```
I expected to be able to use ```s.SiteId``` and ```s.SiteUrl```, but these are undefined.

Stepping through the code using the VSCode debugger, I can see an object ```s.d.Create``` which has those properties with the values I expected to see, but I _really_ expected the Promise to return an ```ISPSiteCreationResponse``` object.

I'm new to both TypeScript _and_ also new to pnpjs, so I could just be doing it wrong. When I try and do a ```console.log(s.d)```, for example, it gives an error “Property 'd' does not exist on type 'ISPSiteCreationResponse'.”

The site is indeed created on our SharePoint Online tenant so that part is working. I just need to get the ```SiteId``` and ```SiteURL``` back to do some further processing.

**Comment by patrick-rodgers** [comment]
Thanks for reporting this, missed it in the list of issues. Will have a look, thanks!

**Comment by patrick-rodgers** [comment]
Fixed this, will be in 1.3.6.
